packages/web-components/development/index.html

```diff
@@ -31,35 +31,50 @@
     --relewise-mobile-grid-template-columns: repeat(1, 1fr);
```

Comment:
Jeg kunne godt tænke mig vi snakkede lidt om dev oplevelsen her på et tidspunkt :)

Det er lidt svært for andre at komme igang her.

Reply:
La' os lige tage snakked når du har tid idag. Det har også irriteret mig en del. 🙈